parsers/bclcontactsparser.py

```diff
@@ -0,0 +1,21 @@
+from utils.exceptions import InvalidFormat
+from operator import itemgetter
+
+
+def BclcontactsParser(input):
+    contents = input.split('\n')
+    output = []
+
+    for line in contents:
+        line = line.lstrip().split()
+        if not line or line[0].isalpha():
+            continue
+        elif line[0].isdigit() and line[2].isdigit() and len(line) >= 5:
```

Comment:
Here you should be checking `len(line)>=10` since you want to use `line[9]` later on.